sql/expression/div.go

```diff
@@ -356,6 +316,38 @@ func (d *Div) floatOrDecimal() sql.Type {
 	return defType
 }
 
+// floatOrDecimalType returns value converted to either float or decimaltype.
```

Comment:
super minor typo ;-) 

```suggestion
// floatOrDecimalValue returns value converted to either float or decimaltype.
```